adala/runtimes/_litellm.py

```diff
@@ -342,6 +354,33 @@ def check_concurrency(cls, value) -> int:
             )
         return value
 
+    @property
```

Comment:
can we please not duplicate this function in 2 places? move it out and have the 2 classes call the same function or something

Reply:
done